dockerfiles/zesty/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:precise
+FROM ubuntu:zesty
```

Comment:
Is that sufficient for removing it here, or do you still want to keep claiming it's supported? :innocent: